### DISCUSSION TITLE
Price Comparison Tool

### DISCUSSION THREAD

**Discussion by ericsilvertx** [CMSgov/price-transparency-guide#161] [General]
Am I correct that the Price Comparison Tool needs to also be available in January 22.  Is it correct that this tool would basically use the data from the three files to allow users to search services and product available to a customer by select the plan they currently have.

If it is then since the data files are the same format is there perhaps someone who has identified a open source web system to query and display the results of a query to see the cost for a procedure across all the doctors that are available in and out of their network?

**Comment by nicholas-dahl-CTS** [comment]
The Transparency in Coverage Cost Estimate Tool requirements go into effect 1/1/2023 and 1/1/2024.  However there is some ambiguity as to the separate and distinct requirements contained in section 114 of the NSA, which refer to the maintenance of a "price comparison tool".  As of today, it is my understanding that these NSA requirements are expected to be implemented by 1/1/2022 on the basis of a good faith interpretation of the legislation, in lieu of having more detailed rules to work from.  In the future, if/when rules are published addressing section 114, different timelines may be specified in order to give a reasonable implementation timeline to plans.  

Separately. I'd like to address the use of MR file data with a price comparison tool.  The MR Files will not have any member specific data in them, and therefore they would likely not be sufficient on their own to be used in a tool that allows already-enrolled members to generate member-specific estimates of care.  Rather, the MR file data would likely be used for industry research and analysis, and could be used by 3rd party app developers to create a tool that allows prospective enrollees to shop for plans based on rate data from the MR files.  This is not a required use case for plans but rather an expected outcome by the industry.  All of this is just my own interpretation, very much open to comments and other thoughts.

**Reply by ericsilvertx** [reply]
The Third-Party area will be where the action is for sure.  However, as an employee with coverage from my employee should It not be easy to click on a link and provide a non-logged-in user to see what cost option within the employer plan there is for a type of service?  The Price Comparison tool will probably be the basis for the full-fledge Cost Estimate Tool to come a year later.
